src/tribler/gui/dialogs/startdownloaddialog.py

```diff
@@ -275,6 +275,12 @@ def on_download_clicked(self, checked):
             )
         else:
             download_dir = self.dialog_widget.destination_input.currentText()
+            self.logger.info(f'Download folder: {download_dir}')
+            if not download_dir:
+                text = tr("Please specify the path to the download directory")
+                title = tr("The download directory is empty")
```

Comment:
The `text` is good, but the `title` is a bit confusing. It can be read as "the download directory exists but contain no files", while the intended meaning is "the path to the download directory is not specified". I think it is better to reword it.

Reply:
Changed to "The path is not specified"